.github/workflows/main.yml

```diff
@@ -167,6 +167,15 @@ jobs:
       - name: git checkout
         uses: actions/checkout@v3
 
+      - name: git checkout fuzzing seeds
+        uses: actions/checkout@v3
+        with:
+          repository: lightninglabs/lnd-fuzz
+          path: lnd-fuzz
+
+      - name: rsync fuzzing seeds
+        run: rsync -a --ignore-existing lnd-fuzz/ ./
```

Comment:
How does this cause to actually run the fuzz tests? Don't we also need to add the call to `make fuzz`?

Reply:
`go test` automatically runs fuzz tests on seeds provided in `/testdata/fuzz`.  No fuzzing, just running the test once per input.

Since `make unit` runs `go test` under the hood, all we need is to copy the seeds to the appropriate place, which is done by `rsync`.